sdk/datashare/Microsoft.Azure.Management.DataShare/src/Generated/Models/ShareSubscription.cs

```diff
@@ -33,41 +33,50 @@ public ShareSubscription()
         /// <param name="type">Type of the azure resource</param>
         /// <param name="createdAt">Time at which the share subscription was
         /// created.</param>
-        /// <param name="createdBy">The user who created the share
-        /// subscription.</param>
+        /// <param name="providerEmail">Email of the provider who created the
+        /// resource</param>
+        /// <param name="providerName">Name of the provider who created the
+        /// resource</param>
+        /// <param name="providerTenantName">Tenant name of the provider who
+        /// created the resource</param>
         /// <param name="provisioningState">Provisioning state of the share
         /// subscription. Possible values include: 'Succeeded', 'Creating',
         /// 'Deleting', 'Moving', 'Failed'</param>
         /// <param name="shareDescription">Description of share</param>
         /// <param name="shareKind">Kind of share. Possible values include:
-        /// 'CopyBased'</param>
+        /// 'CopyBased', 'InPlace'</param>
         /// <param name="shareName">Name of the share</param>
-        /// <param name="shareSender">Sender of the share</param>
-        /// <param name="shareSenderCompanyName">Company name of the share
-        /// sender</param>
         /// <param name="shareSubscriptionStatus">Gets the current status of
         /// share subscription. Possible values include: 'Active', 'Revoked',
         /// 'SourceDeleted', 'Revoking'</param>
         /// <param name="shareTerms">Terms of a share</param>
-        public ShareSubscription(string invitationId, string id = default(string), string name = default(string), string type = default(string), System.DateTime? createdAt = default(System.DateTime?), string createdBy = default(string), string provisioningState = default(string), string shareDescription = default(string), string shareKind = default(string), string shareName = default(string), string shareSender = default(string), string shareSenderCompanyName = default(string), string shareSubscriptionStatus = default(string), string shareTerms = default(string))
+        /// <param name="userEmail">Email of the user who created the
+        /// resource</param>
+        /// <param name="userName">Name of the user who created the
+        /// resource</param>
+        public ShareSubscription(string invitationId, string sourceShareLocation, string id = default(string), string name = default(string), string type = default(string), System.DateTime? createdAt = default(System.DateTime?), string providerEmail = default(string), string providerName = default(string), string providerTenantName = default(string), string provisioningState = default(string), string shareDescription = default(string), string shareKind = default(string), string shareName = default(string), string shareSubscriptionStatus = default(string), string shareTerms = default(string), string userEmail = default(string), string userName = default(string))
```

Comment:
You are making a breaking change to the public api by adding a mandatory parameter. That's why the test fails.

Please consider if this breaking change is what you want. If it is, update the tests and re-record them; if not update swagger spec and re-generate the SDK.

Reply:
I've reverted and regenerated from the latest swagger for now.